docs/source/reference/projections.rst

```diff
@@ -60,7 +59,7 @@ AzimuthalEquidistant
     import matplotlib.pyplot as plt
     import cartopy.crs as ccrs
 
-    plt.figure(figsize=(3, 3))
+    plt.figure(figsize=(3.0101, 3))
```

Comment:
Where did this value come from? Seems like it was OK on the current docs: https://scitools.org.uk/cartopy/docs/latest/reference/projections.html#azimuthalequidistant

Same with the Orthographic update.

Reply:
Struck me as kinda odd too.

I simply done a `make projection` and this is the automated change

Reply:
Did you want me to revert that particular auto-change?

Reply:
Ahh, the width is taken from the `x_limits` of each projection, so my guess is this has to do with PROJ updating some of their limits with a version bump at some point...